src/config/generator.js

```diff
@@ -3,18 +3,21 @@ import path from 'path'
 import fs from 'fs-extra'
 import util from 'util'
 import prettier from 'prettier'
+import chalk from 'chalk'
 
 const prompts = {
-  isSinglePageApp: `\rAre you strapping a single page app? [y/n]:  `,
-  bundlePath: `  Please provide the path to your bundle file [relative to current dir]:  `,
-  bundleHost: `  Please specify the html container your bundle expects [i.e. #container or .host]:  `
+  isSinglePageApp: `      Are you strapping a single page app? ${chalk.blue('[y/n]')}:  `,
```

Comment:
these strings and line 17's string are doing the same thing, but the syntax is different.  Lets strive for consistency there and make these template strings also do the  chalk`whateverstring{color something}` i really like that syntax

Reply:
Absolutely, overlooked those but changed